r/R/array-data.R

```diff
@@ -44,7 +44,7 @@
 ArrayData <- R6Class("ArrayData",
   inherit = ArrowObject,
   active = list(
-    type = function() ArrayData__get_type(self),
+    type=function() ArrayData__get_type(self),
     length = function() ArrayData__get_length(self),
```

Comment:
This is a weird fix, isn't it? Why is there no spacing in the first line but still spacing in the following lines?

Reply:
It's me trying to trigger a linter error on the CI to check my script has worked - I'll later revert this change once the CI is "successful failing" because of it. (Current failures are due to other things I still need to work out).

Will convert this PR to a draft for now given that's all a bit unclear.